src/styles/projects-summary.less

```diff
@@ -87,11 +87,88 @@
     a {
       color: @color-pf-white;
       &:hover {
-        color: @link-color;
-        text-decoration: underline;
+        color: @landing-side-bar-link-color-hover;
+        text-decoration: none;
       }
     }
   }
+  .services-view {
+    .services-item {
+      flex: 1 1 auto;
+      margin-bottom: 50px;
+      min-height: 120px;
+      padding: 0;
+      &:focus,
+      &:hover {
+        color: @landing-side-bar-link-color-hover;
+        .services-item-name {
+          color: @landing-side-bar-link-color-hover;
+        }
+      }
+      // Number of recent items displayed based on sidebar width at different viewport sizes
+      @media (max-width: @screen-xxs-max) {
+        &:nth-child(n+3) {
+          display: none; // display first 2
+        }
+      }
+      @media (min-width: @screen-xs-min) {
+        &:nth-child(n+4) {
+          display: none; // display first 3
+        }
+      }
+      @media (min-width: @screen-sm-min) and (max-width: @screen-md-max) {
+        &:nth-child(n+3) {
+          display: none; // display first 2
+        }
+      }
+      @media (min-width: @screen-lg-min) {
+        &:nth-child(n+4) {
+          display: none; // display first 3
+        }
+      }
+      @media (min-width: @screen-xl-min) {
+        &:nth-child(n+3) {
+          display: flex; // display first 4
+        }
+      }
+      .fa-cubes, .icon-wildfly {
+        position: relative;
+        right: -1px;
+      }
+      .icon-nodejs {
+        position: relative;
+        top: 1px;
+      }
+      .icon-ruby, .fa-question {
```

Comment:
The sidebar icons are smaller and with white circle some looked slightly off center. The  ? and ruby looked slightly overcorrected so I adjusted those 1px. 
<img width="595" alt="screen shot 2017-05-02 at 1 46 06 pm" src="https://cloud.githubusercontent.com/assets/1874151/25631355/269cfe2e-2f3e-11e7-866a-25744df97d30.png">

Wildfly is actually off center in both places so I moved it's rule to services-view.less
<img width="120" alt="screen shot 2017-05-02 at 1 25 28 pm" src="https://cloud.githubusercontent.com/assets/1874151/25631585/d93aba44-2f3e-11e7-8681-a49b56e1e32a.png">
<img width="312" alt="screen shot 2017-05-02 at 1 54 36 pm" src="https://cloud.githubusercontent.com/assets/1874151/25631643/028232f6-2f3f-11e7-96de-4d2bf89cabea.png">

**Adjusted**
<img width="308" alt="screen shot 2017-05-02 at 1 56 37 pm" src="https://cloud.githubusercontent.com/assets/1874151/25631727/3f3a8978-2f3f-11e7-8801-64c15001a9b6.png">
<img width="111" alt="screen shot 2017-05-02 at 1 56 25 pm" src="https://cloud.githubusercontent.com/assets/1874151/25631728/3f458e22-2f3f-11e7-8e1a-c6d695a5e71e.png">